mmr/runner.rs

```diff
@@ -30,26 +30,45 @@ impl Default for Runner {
 }
 
 impl Runner {
+    /// MMR size to last leaf `O(log2(log2(n)))`
+    pub fn mmr_size_to_last_leaf(mmr_size: i64) -> i64 {
+        if mmr_size == 0 {
+            return 0;
+        }
+
+        let mut m = (mmr_size as f64).log2().round() as i64;
+        loop {
+            match (2 * m - m.count_ones() as i64).cmp(&mmr_size) {
+                Ordering::Equal => return m - 1,
+                Ordering::Greater => m -= 1,
+                Ordering::Less => m += 1,
+            }
+        }
+    }
```

Comment:
`last_leaf` to `mmr_size`, enumerating leaves from log2(mmr_size), around `log2n /2`

Reply:
Add tests